health.go

```diff
@@ -487,6 +489,33 @@ func readIntFromFile(filePath string) (num int, err error) {
 	return strconv.Atoi(strings.TrimSpace(string(data)))
 }
 
+func getTHPConfigs() map[string]interface{} {
+	configs := map[string]interface{}{}
+	data, err := os.ReadFile("/sys/kernel/mm/transparent_hugepage/enabled")
+	if err != nil {
+		return map[string]interface{}{"error": err.Error()}
```

Comment:
Is there a reason to fail completely if one of them can't be read? Why no just add the error to the returned stuct...
```suggestion
		configs["enabled_error"] = err.Error()
```
... etc.

Reply:
ok, will change to not fail on error.

Reply:
this is done.